qiita_pet/handlers/study_handlers/artifact.py

```diff
@@ -107,6 +107,16 @@ def patch(self):
         self.write(response)
 
 
+class ArtifactGetSamples(BaseHandler):
+    @authenticated
+    def get(self):
+        aids = map(int, self.request.arguments.get('ids[]', []))
```

Comment:
No need to change. I used `self.get_arguments` heavily on labadmin and I was wondering if you knew about an issue that I didn't.